Baidu Scholar.js

```diff
@@ -87,12 +87,16 @@ function scrape(doc, url) {
 	var diversion = attr(doc, 'i.reqdata', 'diversion');
 	var sign = attr(doc, 'a.sc_q' , 'data-sign');
 	var risUrl = "http://xueshu.baidu.com/u/citation?&url=" + encodeURIComponent(dataUrl) + "&sign=" + sign + "&diversion=" + diversion + "&t=ris";
+	var title = doc.title.replace('_百度学术', '')
```

Comment:
is this used anywhere? If not, delete

Reply:
There are cases RIS file is invalid, this is an insurance.